packages/schemas/src/types/domain.ts

```diff
@@ -0,0 +1,21 @@
+import { type z } from 'zod';
+
+import { Domains } from '../db-entries/index.js';
+
+export const domainSelectFields = Object.freeze([
+  'id',
+  'domain',
+  'status',
+  'errorMessage',
+  'dnsRecords',
+] as const);
+
+export const domainResponseGuard = Domains.guard.pick({
+  id: true,
+  domain: true,
+  status: true,
+  errorMessage: true,
+  dnsRecords: true,
+});
```

Comment:
nit: is there a way we could keep domainSelectFields and domainResponseGuard from a SSOT?

Reply:
try to use this code but failed with response guard:
```
export const domainResponseGuard = Domains.guard.pick(
  Object.fromEntries(domainSelectFields.map((key) => [key, true]))
);
``

created LOG-6236 to do it later